### ISSUE TITLE
draft a story

### ISSUE THREAD

**Issue by lazyhackerthani** [bernie-walker/express#16]
> _Partially dependent on story_ #28 

As a _user_
I want to _draft a story_
So that _I can save my story without publishing it_

**Business Context**
> Problem : Need to draft a story
> Solution : Providing with save only option 

**In Scope**
- save document without publishing it

**Out of Scope**
- auto save
- editing published

**Acceptance Criteria**
- [x] Criterion 1
- Given the user is on the dashboard
- When the user clicks on new story 
- then editor should open with a new untitled story (story should get created with a title 'Untitled Story')

- [x] Criterion 2
- Given the story has a title named _'Untitled Story'_
- When the user opens the story in editor 
- Then the story should open with no text in title bar

- [x] Criterion 3
- Given a user is on the the edit page
- When the user clicks on Draft option
- Then the story should get drafted and appear in Drafted section of user stories page

- [x] Criterion 4
- Given a user is on the _user stories_ page
- When the user clicks on the drafted story
- Then the story should be opened in an editor
